README.md

```diff
@@ -74,7 +74,7 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- [Live Demo Link](https://google.com)
+- Coming soon..
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
```

Comment:
![demo](https://github.com/ayaderaghul/microverse-portfolio/assets/99656687/818c5c7b-fb6b-41c9-8fe2-41fbf87fac7c)


- [ ] If your project is not deployed, kindly delete this section 👍🏾